### DISCUSSION TITLE
custom FBW version same as Dev (Installer)

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#4090] [Q&A] [answered]
hello, frist I want to state that this is only a question, and not a negative annotation or somethink like that! I´m VERY VERY thankful to the whole team for doing such an awesome "job" on this project...
But I have a question concerning the FBW branch and the Installer:

The installer said, that the custom FBW-version is the same as the development version, only including the custom FBW. But i recognize, that the development branch are updated nearly every day, and the FBW version is now around 5 days "old" and do not include the newest features.

Is it an "installer-thing" or is it because some features do not work with the custom FBW, so they were left out?

again, THANK YOU all for what you are doing here for an awesome sim-experience!!! :) :)

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
The custom FBW branch is manually updated by Andreas Guther I believe. He recently left the branch slightly behind but it will be updated soon. Keep in mind that it is called experimental for a reason right?

**Reply by andrejetcat** [reply]
thank you for this information! :) I just wonder. Everything is fine :)

**Comment by hotshotp** [comment]
I think AGuther said that there were some conflicts that he is working on eliminating before rebasing the branch

**Reply by andrejetcat** [reply]
thank you  :) we´ll have to wait :)
